2024-12-11 00:18:59,998 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Current SDK version is 0.19.0
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Configure stats pid to 59400
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Loading settings from /home/seppe/.config/wandb/settings
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Loading settings from /home/seppe/RL/RL_MARL_QMIX/wandb/settings
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Loading settings from environment variables
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_init.py:_log_setup():528] Logging user logs to /home/seppe/RL/RL_MARL_QMIX/wandb/run-20241211_001859-__20241211_001859/logs/debug.log
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_init.py:_log_setup():529] Logging internal logs to /home/seppe/RL/RL_MARL_QMIX/wandb/run-20241211_001859-__20241211_001859/logs/debug-internal.log
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_init.py:init():639] calling init triggers
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_init.py:init():645] wandb.init called with sweep_config: {}
config: {}
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_init.py:init():688] starting backend
2024-12-11 00:18:59,999 INFO    MainThread:59400 [wandb_init.py:init():692] sending inform_init request
2024-12-11 00:19:00,001 INFO    MainThread:59400 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-12-11 00:19:00,002 INFO    MainThread:59400 [wandb_init.py:init():705] backend started and connected
2024-12-11 00:19:00,013 INFO    MainThread:59400 [wandb_run.py:_label_probe_notebook():1327] probe notebook
2024-12-11 00:19:00,027 INFO    MainThread:59400 [wandb_init.py:init():798] updated telemetry
2024-12-11 00:19:00,035 INFO    MainThread:59400 [wandb_init.py:init():830] communicating run to backend with 120.0 second timeout
2024-12-11 00:20:57,387 ERROR   MainThread:59400 [wandb_init.py:init():857] encountered error: Run initialization has timed out after 120.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
2024-12-11 00:20:57,388 ERROR   MainThread:59400 [wandb_init.py:init():1315] error in wandb.init()
Traceback (most recent call last):
  File "/home/seppe/miniconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1305, in init
    return wi.init()
           ^^^^^^^^^
  File "/home/seppe/miniconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 863, in init
    raise error
wandb.errors.errors.CommError: Run initialization has timed out after 120.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Current SDK version is 0.19.0
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Configure stats pid to 59400
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Loading settings from /home/seppe/.config/wandb/settings
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Loading settings from /home/seppe/RL/RL_MARL_QMIX/wandb/settings
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_setup.py:_flush():68] Loading settings from environment variables
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_init.py:_log_setup():528] Logging user logs to /home/seppe/RL/RL_MARL_QMIX/wandb/run-20241211_002157-__20241211_002157/logs/debug.log
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_init.py:_log_setup():529] Logging internal logs to /home/seppe/RL/RL_MARL_QMIX/wandb/run-20241211_002157-__20241211_002157/logs/debug-internal.log
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_init.py:init():639] calling init triggers
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_init.py:init():645] wandb.init called with sweep_config: {}
config: {}
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_init.py:init():688] starting backend
2024-12-11 00:21:57,638 INFO    MainThread:59400 [wandb_init.py:init():692] sending inform_init request
2024-12-11 00:21:57,639 INFO    MainThread:59400 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-12-11 00:21:57,640 INFO    MainThread:59400 [wandb_init.py:init():705] backend started and connected
2024-12-11 00:21:57,648 INFO    MainThread:59400 [wandb_run.py:_label_probe_notebook():1327] probe notebook
2024-12-11 00:21:57,661 INFO    MainThread:59400 [wandb_init.py:init():798] updated telemetry
2024-12-11 00:21:57,668 INFO    MainThread:59400 [wandb_init.py:init():830] communicating run to backend with 90.0 second timeout
2024-12-11 00:23:25,873 ERROR   MainThread:59400 [wandb_init.py:init():857] encountered error: Run initialization has timed out after 90.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
2024-12-11 00:23:25,873 ERROR   MainThread:59400 [wandb_init.py:init():1315] error in wandb.init()
Traceback (most recent call last):
  File "/home/seppe/miniconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 1305, in init
    return wi.init()
           ^^^^^^^^^
  File "/home/seppe/miniconda3/lib/python3.11/site-packages/wandb/sdk/wandb_init.py", line 863, in init
    raise error
wandb.errors.errors.CommError: Run initialization has timed out after 90.0 sec. Please try increasing the timeout with the `init_timeout` setting: `wandb.init(settings=wandb.Settings(init_timeout=120))`.
